nxdrive/data/qml/AccountsTab.qml

```diff
@@ -132,6 +132,26 @@ Rectangle {
                     onClicked: api.open_local(accountSelect.getRole("uid"), "/")
                 }
 
+                // Remaining space
+                ScaledText {
+                    text: qsTr("FREE_DISK_SPACE") + tl.tr;
+                    color: mediumGray
+                }
+                ScaledText {
+                    text: api.get_free_disk_space(folder);
+                    color: api.free_disk_space_under_limit(folder) ? "#ff0000": "#008000"
```

Comment:
When you use colors in QML, check first if you can reuse one of https://github.com/nuxeo/nuxeo-drive/blob/e3d4ce0486aca778a861de45cc5fbd2072d3596a/nxdrive/gui/application.py#L354-L365